src/utils/helpers.test.js

```diff
@@ -56,3 +69,123 @@ describe('getUKFormattedEuros', () => {
     expect(result).toBe('€123.45');
   });
 });
+
+describe('getCurrentMonth', () => {
+  afterEach(() => {
+    global.Date.now = realDateNow;
+  });
+
+  test('if the current date is 2022-06-01, it should return "June 2022"', () => {
```

Comment:
these tests are very clear and nice. I personally prefer the title of the test is more "what" we are testing and not "how"/"what we compare". in this case for example we test "the correct formatting of the month and year". what do you think?

Reply:
Sure thing. I also like when the test statements read like requirements (especially if the requirements are the starting point). Maybe we could go with something like "it should return the current month in "Month YYYY" format, and then we group the comparisons together in the same `it()` or `test()` block?

```JavaScript
it('should return the current month in "Month YYYY" format', () => {
  global.Date.now = getDateNowStub('2022-06-01');
  expect(getCurrentMonth()).toBe('June 2022');
  
  global.Date.now = getDateNowStub('2024-02-29');
  expect(getCurrentMonth()).toBe('February 2024');
  
  // ...
});
```

Reply:
Btw, I imagine we could take the same approach with the "if passed X, should return Y" tests for `getUKFormattedEuros`.

Reply:
great suggestion. i think every line we can save and every thought we can save on the readers part is appreciated.